### ISSUE TITLE
Does not work with ember-cli-sass

### ISSUE THREAD

**Issue by mellatone** [cibernox/ember-basic-dropdown#75]
Created a new cli project with the following devDependencies:
```
  "ember-basic-dropdown": "0.11.0",
  "ember-cli-sass": "5.3.1"
```

and `ember-basic-dropdown` is no longer able to calculate the proper coordinates.

The sample usage from the README, for example yields the following:

```
<div id="ember-basic-dropdown-content-ember383" class="ember-basic-dropdown-content ember-basic-dropdown--transitioning-in  ember-basic-dropdown--below ember-basic-dropdown--left" style="width: 1447px; top: 44px; left: 8px;">
        Content of the dropdown once it appears


</div>
```

Note the style attribute: `style="width: 1447px; top: 44px; left: 8px;"`

**Comment by mellatone** [comment]
Was this reproduce-able?

**Comment by cibernox** [comment]
I tried, it works good for me. I don't think it has anything to do with `ember-cli-sass` but for the snipped you posted, the fact that the dropdown has the class `ember-basic-dropdown--transitioning-in` makes me think that there is some animation or transition going on that never finishes?

I'd be glad to help finding the problem. 

This is the repo where I tried to reproduce it: https://github.com/cibernox/basic-dropdown-dummy-app

**Comment by mellatone** [comment]
Thanks for the sample repo. The dependencies there are good 👍

Unfortunately I'm not on the latest Ember because of legacy view support. Here are my project depenencies (+ breaking example) https://github.com/mellatone/basic-dropdown-debugging

**Comment by mellatone** [comment]
Tldr; of that repo is that it's using the blueprint for ember cli 2.3.0 + cli sass + basic drop down.

**Comment by cibernox** [comment]
I'll check this later/tomorrow.

For the record, someone reported some issue with sass too, and I think the
cause is a new version of node-sass that doesn't allow to have a file named
`ember-power-select.scss` next to a folder named `ember-power-select`.



2016-04-19 22:06 GMT+01:00 Steven Valladolid <notifications@github.com>:

**Comment by cibernox** [comment]
@mellatone I just cloned your repo and everything works ok.

You just need to import the styles in your `app.scss` adding this line: `@import 'ember-basic-dropdown';`

**Comment by cibernox** [comment]
If that was the problem  the entire time I certainly need to update the readme to tell people to explicitly import the styles if they are using sass.

**Comment by mellatone** [comment]
@cibernox Thank you! The sass import fixed the positioning issues. As a first time user, I didn't realize that was necessary.

**Comment by cibernox** [comment]
I've update the readme with more detailed install instructions. At the beginning this addon was extracted from ember-power-select and I never cared too much about documenting it properly, but since more and more people is using it on its own I should create some basic docs.
